train_management/templates/train_management/day_planning_list.html

```diff
@@ -0,0 +1,65 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block content %}
+<h1>{% translate "Day Planning" %}</h1>
+
+<button id="button" class="btn btn-secondary">getSelections</button>
```

Comment:
make label of button translatable
also, since the button doesn't really do anything yet, please deactivate it for the moment